docs/external/faq.md

```diff
@@ -88,6 +88,9 @@ e.g.
 systemctl restart docker
 ```
 
+#### I'm not able to create volume and I see "VolumeDriver.Create: Device not found" error
+
+From 0.19 release of vDVS, the plugin has to have a ESX service thats at least 0.19 or later. This applies for plugins on Linux platform. [#2023](https://github.com/vmware/docker-volume-vsphere/issues/2023)
```

Comment:
`ESX service thats at least 0.19` => `ESX service at least 0.19` or `thats` -> `that's` What do you mean by `thats`?

Reply:
Could mention VIB (use VIB name with the version) perhaps vs. ESX service. 

For Windows the device identification in the plugin is the same as it was, while for Linux the device identification has been upgraded to detect devices on secondary PCI buses.